Legacy/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>5.0.0</string>
+	<string>5.1.0</string>
```

Comment:
```suggestion
	<string>8.0.0</string>
```

I believe last tag was 7.0.4 and plist version was outdated.

Also I don't know if major version should be updated, because request authorizer is now backwards incompatible. @bealex what do you think?

Reply:
I dont know do we use semver or not. If we so then yes, this is major change unless we make this backwards compatible.